pkg/api/configsync/v1alpha1/zz_generated.conversion.go

```diff
@@ -138,6 +138,11 @@ func RegisterConversions(s *runtime.Scheme) error {
 	}); err != nil {
 		return err
 	}
+	if err := s.AddGeneratedConversionFunc((*v1beta1.OverrideSpec)(nil), (*OverrideSpec)(nil), func(a, b interface{}, scope conversion.Scope) error {
```

Comment:
why is there still an OverrideSpec here? should it be RootSyncOverrideSpec or RepoSyncOverrideSpec?

Reply:
oh, i see. you added a shared parent object for embedding. i get it.